src/framework/piral-core/src/components/DefaultLayout.test.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { DefaultLayout } from './DefaultLayout';
+import { mount } from 'enzyme';
+
+describe('Default Layout Component', () => {
+    it('renders correctly for desktop', () => {
```

Comment:
Here I see 4 spaces - it should be 2. Please run prettier as suggested!

Reply:
I ran it, but there was no effect on my document.
Still wondering ...

Reply:
Because you have a syntax error. Prettier only works if your document contains valid syntax.